src/modules/Accordion/AccordionContent.js

```diff
@@ -1,38 +1,34 @@
 import React, { PropTypes } from 'react'
 import cx from 'classnames'
 
+import Content from '../../parts/Content'
```

Comment:
This is an example implementation of a reusable component part, created by the factory.